### DISCUSSION TITLE
HW_4 , part 3

### DISCUSSION THREAD

**Discussion by JordanAlnaimi** [cooperrc/computational-mechanics#59] [Q&A]
I was wondering if anyone knows how to load the data into the file
![Screenshot 2022-06-23 231452](https://user-images.githubusercontent.com/106566966/175652088-4b8deba9-98d8-4534-8c88-5360782b843c.png)

**Comment by Aarons-Joshua** [comment]
Remember to put the skiprows = 1 into the np.loadtxt function: 
xy_data= np.loadtxt('../data/stress_relax.dat',delimiter=',',skiprows=1)

Then you have to assign the rows to their specific variables:
A = xy_data[:,0] 
B= xy_data[:,1] 

Hope this helps

**Reply by Aarons-Joshua** [reply]
I should have explained. The top row in the file, if I am correct, is a string as it is probably empty. You must then skip the top row and only take values out below the first row.

**Reply by JordanAlnaimi** [reply]
Thank you, the code work for me know.
